docs/thehive/operations/backup-restore/backup/hot-backup/hot-backup-cluster.md

```diff
@@ -0,0 +1,96 @@
+# How to Perform a Hot Backup on a Cluster
+
+This topic provides step-by-step instructions for performing a hot backup on a cluster for TheHive.
+
+{!includes/prerequisites-hot-backup-restore.md!}
+
+{!includes/data-consistency-hot-backup.md!}
+
+{!includes/backup-restore-best-practices.md!}
+
+The process requires backing up data from all three components: Apache Cassandra distributed across three nodes, Elasticsearch and file storage.
+
+* [Database backup](#create-cassandra-snapshots)
+* [Indexing backup](#create-elasticsearch-snapshots)
+* [File storage backup](#perform-a-backup-on-file-storage)
+
+## Prerequisites
+
+### Install required tools
+
+{!includes/hot-backup-required-tools.md!}
+
+### Configure systems
+
+{!includes/hot-backup-configure-systems.md!}
+
+<!-- + add MinIO option -->
+
+### Perform preliminary checks
+
+{!includes/preliminary-checks-hot-backup.md!}
+
+### Replicate data across all three nodes
+
+!!! warning "Data replication requirement"
+    If this requirement isn't met, cluster restoration may fail, and integrity issues could arise. It's your responsibility to ensure data replication across all nodes before proceeding.
+
+Before proceeding with the backup, replicate 100% of your data across all nodes. This simplifies the snapshot procedure, allowing snapshots to be taken from just one node.
```

Comment:
Should we insist on the fact this should be done for both Cassandra and ElasticSearch? Something like:
```suggestion
Before proceeding with the backup, replicate 100% of your Cassandra and ElasticSearch data across all nodes. This simplifies the snapshot procedure, allowing snapshots to be taken from just one node.
```